assets/hip-1056/protobuf/stream/output/transaction_output.proto

```diff
@@ -0,0 +1,159 @@
+ * in the original transaction nor present in state changes.
+ *
+ * > Note
+ * >> Only a few transactions produce output that is not in the transaction
+ * >> and also not reflected in state changes. All other transaction types
+ * >> are _currently_ not included here. We have, however, allocated names
+ * >> and indexes for those transaction types to preserve consistency if we
+ * >> add them later.
+ *
+ * <!--
+ * Reserved definitions:
+ * import "stream/output/consensus_service.proto";
+ *    SubmitMessageOutput submit_message;
+ *
+ * import "stream/smart_contract_service.proto";
+ *    UpdateContractOutput contract_update;
+ *    DeleteContractOutput contract_delete;
+ *    SystemDeleteContractOutput
+ *    SystemUnDeleteContractOutput
+ *    CreateTopicOutput create_topic;
+ *    UpdateTopicOutput update_topic;
+ *
+ * import "stream/file_service.proto";
+ *    CreateFileOutput file_create;
+ *    AppendFileOutput file_append;
+ *    UpdateFileOutput file_update;
+ *    DeleteFileOutput file_delete;
+ *    SystemDeleteOutput system_delete;
+ *    SystemUndeleteOutput system_undelete;
+ *
+ * import "stream/crypto_service.proto";
+ *    UpdateNodeStakeOutput update_node_stake;
+ *    ApproveAllowanceOutput approve_allowance;
+ *    DeleteAllowanceOutput delete_allowance;
+ *    CreateAccountOutput create_account;
+ *    UpdateAccountOutput update_account;
+ *    DeleteAccountOutput delete_account;
+ *
+ * import "stream/token_service.proto";
+ *    CreateTokenOutput create_token;
+ *    DeleteTokenOutput delete_token;
+ *    FreezeTokenAccountOutput freeze_token_account;
+ *    UnfreezeTokenAccountOutput unfreeze_token_account;
+ *    GrantTokenKycOutput grant_token_account_kyc;
+ *    RevokeTokenKycOutput revoke_token_account_kyc;
+ *    UpdateTokenOutput update_token;
+ *    UpdateTokenNftsOutput update_token_nfts;
+ *    MintTokenOutput mint_token;
+ *    BurnTokenOutput burn_token;
+ *    WipeTokenAccountOutput wipe_token_account;
+ *    AssociateTokenOutput associate_token;
+ *    DissociateTokenOutput dissociate_token;
+ *    UpdateTokenFeeScheduleOutput update_token_fee_schedule;
+ *    PauseTokenOutput pause_token;
+ *    UnpauseTokenOutput unpause_token;
+ *
+ * import "stream/consensus_service.proto";
+ *    DeleteTopicOutput delete_topic;
+ *
+ * import "stream/schedule_service.proto";
+ *    DeleteScheduleOutput delete_schedule;
+ *
+ * import "stream/network_service.proto";
+ *    FreezeOutput freeze_network;
+ * -->
+ */
+message TransactionOutput {
+    oneof transaction {
+        /**
+         * Output from a crypto transfer transaction.
+         */
+        CryptoTransferOutput crypto_transfer = 1;
+
+        /**
+         * Output from a utilPrng transaction to request a
+         * deterministic pseudo-random number.
+         */
+        UtilPrngOutput util_prng = 2;
+
+        /**
+         * Output from a contract call transaction.
+         */
+        CallContractOutput contract_call = 3;
```

Comment:
All the outputs are named inconsistently. UtilPrng has the service name first then the operation, but CallContract has the operation first followed by the service name. Then the field names do not match the message type name e.g. CallContract != contract_call.

In my opinion, we should name both the message and the field exactly as the transaction body name prefix with TransactionBody replaced with Output. So if it's `ContractCallTransactionBody` this would be `ContractCallOutput contract_call`.